bundles/org.openhab.binding.modbus.sunspec/src/main/java/org/openhab/binding/modbus/sunspec/internal/handler/AbstractSunSpecHandler.java

```diff
@@ -164,6 +170,26 @@ private void startUp() {
         }
     }
 
+    /**
+     * Load and parse configuration from the properties
+     * These will be set by the auto discovery process
+     */
+    private Optional<ModelBlock> getAddressFromProperties() {
```

Comment:
The use of Optional here would be better substituted by returning a nullable ModelBlock instead.

Reply:
Ok, changed to nullable.